brping/ping1d.py

```diff
@@ -11,6 +11,7 @@
 from brping import pingmessage
 import serial
 import time
+from collections import deque
```

Comment:
Should collections be before brping import ? For import/alphabetic reasons.

Reply:
[Pep8 suggests something else](https://www.python.org/dev/peps/pep-0008/#imports). And by that we are both wrong, I'll re-organize things